### ISSUE TITLE
Technologies

### ISSUE THREAD

**Issue by atran2424** [18F/tts-buy-datagov-technical-support-services#37]
# Question/Comment

## Name and affiliation
Proposal Coordinator

## Section of RFQ documents
Page 15

## Question/Comment
GSA states, “The Offeror should describe how it plans to use CKAN, WordPress, and the other technologies in the Data.gov stack as described in Section 1 to deliver the development support required by Data.gov….” Does this requirement refer to the tools listed in paragraph 2 on page 3 of the solicitation beginning with “The system involves a variety of open source applications”?

**Comment by oghaffari** [comment]
Yes, as well as technologies described in the GitHub repositories referenced.
